### PR TITLE
Translation and other Document updates

### PR DESCRIPTION
**Pull request by kallleeeeh**
I have update some documentation.
- install.md: more details so people that never developed on windows have an easier start
- working_with_kv_files.md: added link to sublime text's package control installation
- townhall_meetings.md: added new meetings
- translation.md: update to transifex workflow

### CODE REVIEW

File: docs/install.md [hunk 1]
```diff
@@ -3,6 +3,7 @@ You have to set up several things before you can run a local instance of OAA for
 
 ## System Dependencies
 Start by installing these programs:
+ * Windows
```
Comment (Chronophylos) [inline docs/install.md]: lol

File: docs/translations.md [hunk 2]
```diff
@@ -1,66 +1,40 @@
...
-Only begin translations when English text is approved. Add in the complete translation before checking the list item.
-
- * [ ] Chinese translation: ""
- * [ ] Russian translation: ""
- * [ ] Spanish translation: ""
- * [ ] Portuguese translation: ""
+There's a lot of text in OAA, varying between single word titles to paragraph descriptions.
+
+## Languages 
+We are currently planning to translate Open Angel Arena to:
```
Comment (Chronophylos) [inline docs/translations.md]: you can remove the planning

File: docs/translations.md [hunk 3]
```diff
@@ -1,66 +1,40 @@
...
+- **Chinese**
+- Czech
+- **German**
+- Hungarian
+- Polish
+- Portuguese
+- **Russian**
+- **Spanish**
+
+Bold languages have been planned from the beginning.
```
Comment (Chronophylos) [inline docs/translations.md]: This information is unimportant. You could write 'These languages are currently translated'.

File: docs/translations.md [hunk 4]
```diff
@@ -1,66 +1,40 @@
...
+
+## transifex
```
Comment (Chronophylos) [inline docs/translations.md]: the `t` is not capitalized

File: docs/translations.md [hunk 5]
```diff
@@ -1,66 +1,40 @@
...
+We are using Transifex to handle all translation work. In order to start translating, you need to:
+- Create an account on [transifex](https://www.transifex.com/) (Use the same name as in Discord)
+- Go to [our project](https://www.transifex.com/open-angel-arena/open-angel-arena/)
+- Click on `Help Translate "Open Angel Arena"` and follow the instructions
+- Wait to be accepted
+
+## Translating
+We want our translations to blend in with the original Dota so you need to learn the "translation style" of your language. For example, item names are not translated in all languages. In that case, we will also keep the original english name.
+Many strings can be copied from the [official Dota 2 website](http://www.dota2.com/items/) (you can select the language in the top right), which is also great to learn the translation style.
+Sometimes it is hard to find out where a string will be used in the game by just reading it. If that happens you can take a look at the key (below the box in which you enter your translation). For example, `DOTA_Tooltip_Ability_item_bfury_2_Description` is the tooltip description of the abilities of the item Battlefury.
```
Comment (Chronophylos) [inline docs/translations.md]: > is the tooltip description of the abilities of the item Battlefury.
aka the active

File: docs/translations.md [hunk 6]
```diff
@@ -1,66 +1,40 @@
...
-   * [ ] **MORE_KEYS**: "russian"
- * [ ] Spanish translation: ""
-   * [ ] **PUT_KEY_HERE**: "spanish"
-   * [ ] **OTHER_KEY_HERE**: "spanish"
-   * [ ] **MORE_KEYS**: "spanish"
- * [ ] Portuguese translation: ""
-   * [ ] **PUT_KEY_HERE**: "portuguese"
-   * [ ] **OTHER_KEY_HERE**: "portuguese"
-   * [ ] **MORE_KEYS**: "portuguese"
+npm install request fs path parse-kv
```
Comment (Chronophylos) [inline docs/translations.md]: `npm install` should be enough. Packages are listed in `/package.json`

File: docs/install.md (hunk 1 above)
Comment (kallleeeeh) [inline docs/install.md]: Added that since it took me an hour to realize that the dev tools are windows only.

**Comment by Chronophylos** [comment]
I fixed your title

**Pull request merged**